README.md

```diff
@@ -1 +1,42 @@
-# test-automatiom-testNG
\ No newline at end of file
+# TestNG example
+
+Application model which describes the behavior of running the test with gradle's help
+
+## Getting Started
+
+## Running
+
+1. Open project;
+2. Select the Run Configuration;
+3. Wait.
+
+## Testing
+
+Please use next rules for test the app:
```

Comment:
- would be better to use "the following" instead of "next"
 - " for testing" sounds much cleaner